docs/source/02_get_started/05_example_project.md

```diff
@@ -55,7 +55,7 @@ For project-specific settings to share across different installations (for examp
 
 The folder contains three files for the example, but you can add others as you require:
 
--   `catalog.yml` - [Configures the Data Catalog](../05_data/01_data_catalog#using-the-data-catalog-within-kedro-configuration) with the file paths and load/save configuration required for different datasets
+-   `catalog.yml` - [Configures the Data Catalog](../05_data/01_data_catalog.html#using-the-data-catalog-within-kedro-configuration) with the file paths and load/save configuration required for different datasets
```

Comment:
```suggestion
-   `catalog.yml` - [Configures the Data Catalog](../05_data/01_data_catalog.md#using-the-data-catalog-within-kedro-configuration) with the file paths and load/save configuration required for different datasets
```
Pretty sure it should point to the Markdown file, and the docs will get generated correctly. See https://github.com/quantumblacklabs/kedro/blame/72ba895d3db8ecd2efae3419b298877138145291/docs/source/05_data/01_data_catalog.md#L266 for an example.

Reply:
I think you're right. There was just no extension there and the link was returning a 404.
Thanks for the correction, though.